resources/pre-upgrade.sh

```diff
@@ -13,7 +13,7 @@ echo Found old version "${version_parts[0]}"-"${version_parts[1]}"
 
 # When the old dogu version is less then 2.0.0-31, we have to remove the old plugin folder
 if [ "${version_parts[0]}" == "2.0.0" ]; then
-  MINOR_VERSION_DELETE_PLUGINS=31
+  MINOR_VERSION_DELETE_PLUGINS=43
```

Comment:
Maybe i get this wrong, but shouldn't it be less than **44**? Or does version 43 also contain the 2.0.0 release?

Reply:
You're right, this would have been bad. Fixed this. #thx